app/routes/dexscreener/v1/asset.mjs

```diff
@@ -0,0 +1,63 @@
+import yesql from "yesql";
+import path from "path";
+import { dirname } from "../../../../variables.mjs";
+import { CACHE_SETTINGS } from "../../../../variables.mjs";
+import { cachedFetch } from "../../../../helpers/cache_helpers.mjs";
+
+const sqlQueries = yesql(path.join(dirname(), "queries/dexscreener/v1/"), {
+  type: "pg",
+});
+
+export default async (fastify, opts) => {
+  fastify.route({
+    url: "/asset/:asset?",
```

Comment:
`Request: GET /asset?id=:string`

According to the spec they expect `id` as a query param, not a path param